### PR TITLE
Added support for the Font::Info struct

### PR DESCRIPTION
**Pull request by zsbzsb**
Better this time?

### CODE REVIEW

File: include/SFML/Graphics/Font.h [hunk 1]
```diff
@@ -32,6 +32,7 @@
 #include <SFML/Graphics/Glyph.h>
 #include <SFML/Graphics/Types.h>
 #include <SFML/System/InputStream.h>
+#include <SFML/Graphics/FontInfo.h>
```
Comment (LaurentGomila) [inline include/SFML/Graphics/Font.h]: I like when all includes are grouped by module :neckbeard:

**Comment by LaurentGomila** [comment]
Nice, It took me long to find something to say about this PR :smile_cat:

**Comment by zsbzsb** [comment]
>Nice, It took me long to find something to say about this PR

Hahaha :smile: 

...and fixed

**Comment by LaurentGomila** [comment]
Ooooh, alphabetical order. I'm starting to like you :heart_eyes:

**Pull request merged**